torch/fx/experimental/sym_node.py

```diff
@@ -793,6 +804,14 @@ def binary_magic_impl(self, other):
                 self, handle_sym_dispatch(op, (wrap_node(self), wrap_node(other)), {})
             )
         assert isinstance(other, SymNode)
+
+        # Sympy will incorrectly conclude that Mod(s0, 2.0) can never equal
+        # 0.0.  So if we attempt to do a modulus with float, just force
+        # specialization and stop trying to symbolically reason about it
+        if method == "mod" and (self.pytype is float or other.pytype is float):
+            out = self.guard("", 0) % other.guard("", 0)
```

Comment:
Why the empty file / line_no, instead of using the the file/line_no where the mod call came from?